templates/emails/group.monthlyreport.hbs

```diff
@@ -56,10 +56,10 @@ th {
 </div>
 
 <div>
-  <p><i>Previous email didn't include all the transactions happening on the very last day of the last month (11/30/2016). Here is an updated version of the report.</i></p>
 <p>Hi {{recipient.firstName}}!<br />
-We have lots of good news this month, we could barely wait to share them with you! But first, your monthly stats (visit your collective page to get the latest up to date data: <a href="{{group.publicUrl}}?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">{{group.publicUrl}})</a>.<br />
-And please don't hesitate to send us feedback - just reply to this email! We love to hear from you! ❤️❤️</p>
+We were so happy 2016 had finally ended that we missed sending your December report on time! 🙊 <br/>
```

Comment:
Sounds awkward. "We got so excited about 2016 ending that we missed sending December report on time! 🙊 "